### PR TITLE
feat: onboard olark web device mode

### PR DESCRIPTION
**Pull request by mihir-4116**
## PR Description

Please include a summary of the change along with the relevant motivation and context.

## Notion ticket

Ticket link

## Screenshots

Please add screenshots for any new features or UI bug fixes for the following browsers -

- Chrome
  >
- Firefox
  >
- Safari
  >

## Security

- [ ] The code changed/added as part of this pull request won't create any security issues with how the software is being used.

- - -
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/763)

### CODE REVIEW

**Review by shrouti1507 — DISMISSED** [review]
wait for SDK team approval

**Review by MoumitaM — APPROVED** [review]

**Review by MoumitaM — APPROVED** [review]

**Pull request merged**
